src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -314,8 +314,10 @@ protected function landoUri($siteDir): string
                 );
             }
             // Detect multi-site configurations.
+            // We look for the $siteDir to be at the beginning of the appserver
+            // proxy URL.
             $siteDomains = array_filter($landoCfg['proxy']['appserver'], fn($domain) =>
-                strpos($domain, $siteDir) !== false);
+                strpos($domain, $siteDir) == 0);
```

Comment:
```suggestion
                strpos($domain, $siteDir) === 0);
```
We still want strict comparison right?

Reply:
You beat me to it!